web/server/api/endpoints/plan.py

```diff
@@ -37,6 +37,10 @@ async def initiate_plan(
                 plan_options,
                 environment,
                 plan_dates,
+                exception=ApiException(
+                    message="Unable to run a plan",
+                    origin="API -> plan -> initiate_plan",
+                ),
```

Comment:
What do you think about this pattern instead of passing an exception to run_in_executor?
```
def get_plan_builder(...):
    try:
        _get_plan_builder(...)
    except Exception as e:
        tracker_stage_changes.stop(success=False)
        tracker.stop(success=False)
        api_console.log_event_plan_overview()
        raise ApiException(
            message="Unable to get plan changes",
            origin="API -> plan -> run_plan",
        ) from e

def _get_plan_builder(...):
    # original get_plan_builder function
```